src/components/Authentications/Login.jsx

```diff
@@ -1,36 +1,74 @@
 import React, { Component } from 'react'
+import axios from 'axios'
+import { Redirect } from 'react-router-dom'
 
-export default class Login extends Component {
-    render() {
-        return (
-            <>
-            <form onSubmit={this.handleSubmit} method="post">
-                <input type="email" onChange={this.handleInput} name="email" placeholder="Email address" />
-                <input type="password" onChange={this.handleInput} name="password" placeholder="Password" />
-                <input type="submit" value="Login" />
-            </form>
-            <div className="signup">
-                <p>or</p>
-                <ul>
-                    <p>Sign in with</p>
-                    <li>
-                        <a href="#">
-                            <i className="fab fa-facebook" />
-                        </a>
-                    </li>
-                    <li>
-                        <a href="#">
-                            <i className="fab fa-google" />
-                        </a>
-                    </li>
-                    <li>
-                        <a href="#">
-                            <i className="fab fa-linkedin-in" />
-                        </a>
-                    </li>
-                </ul>
-            </div>
-        </>
-        )
-    }
+const Failed = () => {
+	return (
+		<div className='login-warning'>
+			Login Failed. Please try again.
+		</div>
+	)
 }
+
+class Login extends Component {
+	state = {
+		email: '',
+		password: '',
+		status: ''
+	}
+
+	handleSubmit= async e => {
+		e.preventDefault()
+		try {
+			const res = await axios.post('http://192.168.1.125:3000/mentors/login', this.state)
+			console.log(`im here ${res.message}`)
+			if (res.status == 200) {
+				this.setState({ status: res.status })
+			}
+		} catch (error) {
+			this.setState({status: 409})
+		}
+	}
+
+	handleInput = e => {
+		this.setState({ [e.target.name]: e.target.value })
+	}
+
+	render() {
+		return (
+			<>
+				{this.state.status === 200 && <Redirect to='/home' />}
+				{this.state.status !== '' && this.state.status !== 200 && <Failed />}
```

Comment:
@Prashant-Acharya is it necessary to add ```this.state.status !== ''```. 

i think it validates the ```this.state.status !==200 ``` our case.

Reply:
If we don't check for the empty status value, it shows failed component without the first try too.

The better approach in my opinion is to check for the status code 409 which I will implement now.